cmd/sonictool/db/dbutils.go

```diff
@@ -61,7 +61,7 @@ func MakeDbProducer(chaindataDir string, cacheRatio cachescale.Func) (kvdb.FullD
 	})
 }
 
-func MakeGossipDb(dbs kvdb.FullDBProducer, dataDir string, validatorMode bool, cacheRatio cachescale.Func) (*gossip.Store, error) {
+func MakeGossipDb(dbs kvdb.FullDBProducer, dataDir string, validatorMode bool, cacheRatio cachescale.Func, isFakeNet bool) (*gossip.Store, error) {
```

Comment:
nit: try to avoid boolean flags. They do harm readability on the caller's end significantly since they do not provide any hint what `true` or `false` is supposed to mean.